server/common/app_config.py

```diff
@@ -35,6 +35,15 @@ def __init__(self, **kw):
         self.enable_reembedding = False
         self.anndata_backed = False
 
+        # The index page when in multi-dataset mode:
+        #   None:  this returns a 404 code
+        #   "test":  loads a test index page, which links to the datasets that are available in the dataroot
+        #   <redirect>:  anything else will result in a redirect:  flask.redirect(config.dataroot_index)
+        self.multi_dataset_index = "test"
```

Comment:
to make this more "clear", you might want to use a True/False/URL value -- ie,
- False or None - 404
- True - generate index
- string/URL - redirect
This is a nit.

Reply:
Done